src/components/DiscountCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import { View, Text, TouchableOpacity } from "react-native";
+import { FontAwesome } from "@expo/vector-icons";
+
+interface DiscountCardProps {
+  title: string;
+  description: string;
+}
+
+const DiscountCard = ({ title, description }: DiscountCardProps) => {
+  const [isClicked, setisClicked] = useState(false);
+  return (
+    <TouchableOpacity
+      className="pb-2 mb-8 border-b border-b-gray-500"
+      onPress={() => setisClicked(!isClicked)}
```

Comment:
gosto de usar o setter assim quando estou mutanto o valor do state anterior

```suggestion
      onPress={() => setIsClicked(v => !v)}
```